jenkins/opensearch/distribution-build.jenkinsfile

```diff
@@ -140,6 +140,32 @@ pipeline {
                         }
                     }
                 }
+                stage('macos-build-x64-tar') {
+                    agent {
+                        node {
+                            label 'Jenkins-Agent-MacOS12-X64-Mac1Metal-Multi-Host'
+                        }
+                    }
+                    tools {
+                        jdk dockerAgent.javaVersion
+                    }
```

Comment:
Is it something specially needed by macos? Because each docker image has java version specified in manifest args that is used for building the artifact

Reply:
Hey @gaiksaya, we are not going with docker in mac, there is no docker installed, docker cli for mac is not supported, hence it has to be done this way.